.github/workflows/develop.yml

```
@@ -26,3 +26,31 @@ jobs:
       run: docker push islasgeci/dummy_transformations:latest
     - name: Sube sha a Docker Hub
       run: docker push islasgeci/dummy_transformations:${GITHUB_SHA:0:4}
+  build-n-publish:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI
+    runs-on: ubuntu-18.04
```

¿Funcionará con la imagen más nueva?

```suggestion
    runs-on: ubuntu-latest
```